samples/matter/light_bulb/src/app_task.cpp

```diff
@@ -428,3 +279,56 @@ void AppTask::UpdateClusterState()
 		}
 	});
 }
+
+CHIP_ERROR AppTask::Init()
+{
+	/* Initialize Matter stack */
+	ReturnErrorOnFailure(
+		Nordic::Matter::InitServer(MatterEventHandler, Nordic::Matter::InitData{ .mPostServerInitClbk = [] {
+						   app::SetAttributePersistenceProvider(&gDeferredAttributePersister);
+						   return CHIP_NO_ERROR;
+					   } }));
+
+	if (!GetBoard().Init(ButtonEventHandler)) {
+		LOG_ERR("User interface initialization failed.");
+		return CHIP_ERROR_INCORRECT_STATE;
+	}
+
+	int ret{};
+#ifdef CONFIG_AWS_IOT_INTEGRATION
+	ret = aws_iot_integration_register_callback(AWSIntegrationCallback);
+	if (ret) {
+		LOG_ERR("aws_iot_integration_register_callback() failed");
+		return chip::System::MapErrorZephyr(ret);
+	}
+#endif
+
+	/* Initialize trigger effect timer */
+	k_timer_init(&sTriggerEffectTimer, &AppTask::TriggerEffectTimerTimeoutCallback, nullptr);
+
+	/* Initialize lighting device (PWM) */
+	uint8_t minLightLevel = kDefaultMinLevel;
+	Clusters::LevelControl::Attributes::MinLevel::Get(kLightEndpointId, &minLightLevel);
```

Comment:
Access of the attribute store before the Matter data model is initialied

Reply:
Of course I agree, but just to clarify, isn't it the problem also for the current implementation? I mean even without changes from this PR.

Reply:
Yes, it is :). I guess it's not a big deal when loading "constant" attributes. The problem would be more visible with persistent attributes (like the light brightness) as the attributes wouldn't manage to be loaded from flash before `Get`.